### PR TITLE
Fix duplicated costing benchmark

### PR DESCRIPTION
**Pull request by kwxm**
This is a two-character PR to fix a duplicated entry in a list of costing benchmarks.  I just want to fix it before I forget.

Pre-submit checklist:
- Branch
    - [x] Commit sequence broadly makes sense
    - [x] Key commits have useful messages
    - [ ] Relevant tickets are mentioned in commit messages
- PR
    - [x] Self-reviewed the diff
    - [x] Useful pull request description
    - [ ] Reviewer requested
- If you updated any cabal files or added Haskell packages:
    - [ ] `nix-shell shell.nix --run updateMaterialized` to update the materialized Nix files
    - [ ] Update `hie-*.yaml` files if needed
- If you changed any Haskell files:
    - [ ] `nix-shell shell.nix --run fix-stylish-haskell` to fix any formatting issues
- If you changed any Purescript files:
    - [ ] `nix-shell shell.nix --run fix-purty` to fix any formatting issues

Pre-merge checklist:
- [ ] Someone approved it
- [ ] Commits have useful messages
- [ ] Review clarifications made it into the code
- [ ] History is moderately tidy; or going to squash-merge

### CODE REVIEW

**Review by michaelpj — APPROVED** [review]

**Comment by kwxm** [comment]
It seemed to be getting stuck on the cost model test for `verifySignature` so I've temporarily removed that to see what happens.

**Comment by kwxm** [comment]
I should also re-run `plutus-core:plutus-core-budgeting-bench` to regenerate `benching.csv`, which currently doesn't contain any results for `GreaterThanInteger` (but two sets of results for `GreaterThanEqInteger`).   This hasn't shown up previously because `plutus-core:plutus-core-create-cost-model`  reads the csv file and runs R regression methods to obtain linear models which it writes to `costModel.json`, but it re-uses the model for `lessThanInteger` for all of the other integer order comparisons (which is perfectly reasonable since the benchmarks are essentially identical for all of these), so it never attempts to read the stuff that's missing in `benching.csv`.

It's perhaps debatable whether `benching.csv` and the rest should be checked into git since they're figures for a specific machine, but without that everyone would have to run the builtin benchmarks themselves, which takes hours and should only be done when the machine's not doing anything else.  You also need `costModel.json` because `plutus-core-test-cost-model` depends on it.

**Comment by michaelpj** [comment]
> It's perhaps debatable whether benching.csv and the rest should be checked into git since they're figures for a specific machine

Well, I think the idea is that they'll eventually be the "committed" numbers that we're going to use on-chain, which will indeed come from *some* machine where we generate them, but are shared data that we ought to check in.

**Comment by kwxm** [comment]
> > It's perhaps debatable whether benching.csv and the rest should be checked into git since they're figures for a specific machine
> 
> Well, I think the idea is that they'll eventually be the "committed" numbers that we're going to use on-chain, which will indeed come from _some_ machine where we generate them, but are shared data that we ought to check in.

Let me rephrase that a bit.  If anyone runs the cost model benchmarks then it'll update that file, which could then be committed by accident (I keep getting caught out by this); also, `costModel.json` depends on it but isn't automatically regenerated when it changes.  Maybe the benchmarking thing should generate a file with a different name which isn't checked in, and if you really want it to be then you should copy it to the "official" benchmark results file.  I don't really mind though: it's just a minor annoyance.

**Comment by michaelpj** [comment]
> Maybe the benchmarking thing should generate a file with a different name which isn't checked in, and if you really want it to be then you should copy it to the "official" benchmark results file.

That sounds very sensible.

**Comment by kwxm** [comment]
I think I'll merge this when the tests pass to get it out of the way.  It turned out to be more complicated than just removing two characters since I had to re-run the benchmarks for builtins to fill in the missing data, and then regenerate the cost model data in `costModel.json`.

Note that the tests in `plutus-core:cost-model-tests` *fail* with the current version of `costModel.json` because there are negative intercepts in some of the models and the tests check for that.  Fortunately we recently changes those tests into a benchmark because they were already causing timeouts in CI, so the fact that they fail won't cause any problems.  I'm working on the cost model anyway, so I'll fix this problem there.  In the meantime it would be helpful to merge this so that I can work on the cost models in their own branch.  I've added a reminder to fix this in SCP-1994 just to be sure that this does get fixed.
